client/my-sites/plans-features-main/index.jsx

```diff
@@ -362,14 +362,17 @@ export class PlansFeaturesMain extends Component {
 			showTreatmentPlansReorderTest,
 			flowName,
 			is2023PricingGridVisible,
+			plansToShow,
```

Comment:
`getPlansFromProps` above tries to get the plans from `planTypes` first. It's an old piece of code, and it looks like it achieves the same thing. Can we use that prop instead (or remove/rename that one if nothing links to it)? maybe better to remove, cause `plansToShow` has a better semantic than `planTypes`, I think.

Reply:
Ok, good idea, I can iterate on that ✌️

Reply:
> `plansToShow` has a better semantic than `planTypes`

I'm not too sure about this though! Naming is important and should be self-conveying @southp maybe for a vote :-)

Reply:
Great to hear that @southp, what will you combine the two props to? Something like `planTypesToShow` ? Possibilities are endless here

Reply:
Thanks for explaining @southp . Happy with the direction. I guess we prioritize a review on #76070 as a next step and rebase here